js/components/Breakpoints.js

```diff
@@ -1,24 +1,40 @@
 const React = require("react");
 const dom = React.DOM;
+const { connect } = require("react-redux");
+const { getSources } = require("../queries");
 
-function Breakpoints({ breakpoints }) {
+require("./Breakpoints.css");
+require("../lib/variables.css");
 
-  function onResumeClick() {
-    console.log('click')
-    gThreadClient.resume();
-  }
+const Breakpoints = React.createClass({
+
+  render() {
+    function onResumeClick() {
+      console.log('click')
+      gThreadClient.resume();
+    }
 
-  return dom.div(
-    {},
-    dom.button({ onClick: onResumeClick }, 'resume'),
-    dom.ul(
-      null,
-      breakpoints.map(bp => dom.li(
+    function getFilenameFromSources(sources, actor) {
```

Comment:
I wonder if it'd be helpful to move some of the breakpoint source data collection to queries.js. So, instead of getting sources in the breakpoints component and formatting it with `URL`, you could get breakpoint props with a formatted pathname, which would be useful here and maybe other contexts as well.